<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotesCreateWindow</class>
 <widget class="QDialog" name="NotesCreateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>390</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>390</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>390</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="CreateNoteBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>520</width>
     <height>350</height>
    </rect>
   </property>
   <property name="title">
    <string>Create Note</string>
   </property>
   <widget class="QGroupBox" name="TitleBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>Title</string>
    </property>
    <widget class="QPlainTextEdit" name="TitlePlainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ContentBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>120</width>
      <height>110</height>
     </rect>
    </property>
    <property name="title">
     <string>Content</string>
    </property>
    <widget class="QPlainTextEdit" name="ContentPlainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>80</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="CreatedDateBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Created Date</string>
    </property>
    <widget class="QDateEdit" name="CreatedDateEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="TasksBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>350</width>
      <height>260</height>
     </rect>
    </property>
    <property name="title">
     <string>Tasks</string>
    </property>
    <widget class="QTableView" name="TasksTableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>330</width>
       <height>230</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>480</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="CreateNoteButton">
       <property name="text">
        <string>Create Note</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
